src/Network/Network/Models/PSNetworkVirtualApplianceRestartOperationStatusResponse.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
```

Comment:
Please follow this [example](https://github.com/Azure/azure-powershell/blob/8dddb77e2773b9a6778f8e9c91c6a9876d40ae1f/src/Network/Network/Models/PSApplicationGatewayFirewallCustomRuleGroupByUserSession.cs#L2)  to add copyright for newly created files.